elastio-lambda-stream-backup/readme.md

```diff
@@ -0,0 +1,68 @@
+Before starting, please check do you have aws-cliv2, and docker cli installed.
+
+1. From AWS console, select IAM, select Roles then create IAM role with "AdministratorAccess".  
+
+2. From the AWS console, select ECR and create an ECR repository for storing your docker image.
+
+3. Unpack archive and go into unpacked folder.
+# if you want use arm64/amd64 architecture you should use the instance in the same architecture to building image
+
+4. Build and push docker image:
+	# Before starting doing this paragraph look at dockerfile and read comments,
+	# In the comments you can see the possible values for arguments such as ARCH(architecture), VERSION_TAG(elastio cli version)
+	4.1 Register your docker cli by getting the ecr public image with the command:
+	"aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws"
+
+	4.2. Open your ECR repository that we created earlier and press the option "View push commands"
+	If will show you 4 commands that you should run in your terminal that will build your docker image.
+
+5. Create a Lambda function:
+
+	5.1 Go to your AWS console and select the Lambda service, select Create Function, then select "Container image".
+
+	5.2 Enter function name
+
+	5.3 Press button "Browse images" and choose the repository and version of image that you created in section 4.
+
+	5.4 In the Permission paragraph press "Change default execution role" and choose the option "Use an existing role" and
+	in the field below enter the name of the role that we created earlier in the section 1.
+
+	5.5 Press "Create function"
+	
+6. Edit Lambda function configuration:
+
+	6.1 Open "Configuration" tab
+
+	6.2 In paragraph "Execution role" press "Edit"
+
+	6.3 In field "Memory" you should set value "10240"
+
+	6.4 In field "Timeout" you should set up time to 15 min.
+
+	6.5 Press "Save"
+	
+7. Test Lambda
+
+	7.1 Open "Test" tab
+
+	7.2 Put the same json in test field
+
+	{
+	  "bucket_name": "<your-backet-name>",
+	  "file_key": "<file-key>",
+	  "stream_name": "<elastio-stream-name>",
+	  "vault_name": "<vault-name>"
+	}
```

Comment:
All code snippets and CLI command should be formatted as `code`, eg:

```suggestion
	```json
	{
	  "bucket_name": "<your-backet-name>",
	  "file_key": "<file-key>",
	  "stream_name": "<elastio-stream-name>",
	  "vault_name": "<vault-name>"
	}
	```
```